domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1006,6 +1008,23 @@ public void optimizedSqlCanSetSize() {
     assertThat(PgUtil.getOptimizedSqlSize(), is(oldSize));
   }
 
+  @Test
+  public void getTotalRecordsTest() {
+    assertNull(PgUtil.getTotalRecords(10, null, 0, 0));
+
+    assertEquals(PgUtil.getTotalRecords(10, 20, 0, 0),  (Integer)20);
+
+    assertEquals(PgUtil.getTotalRecords(0, 20, 10, 10),  (Integer)10);
```

Comment:
limit and offset should be different to be sure that the result the method takes the offset and not the limit.